src/CodeStyle/VisualBasic/Analyzers/BasicCodeStyle.vbproj

```diff
@@ -10,6 +10,7 @@
     <AssemblyName>Microsoft.CodeAnalysis.VisualBasic.CodeStyle</AssemblyName>
     <TargetFramework>netstandard1.3</TargetFramework>
     <PackageTargetFallback>portable-net45+win8;dotnet</PackageTargetFallback>
+    <NoWarn>$(NoWarn);40057</NoWarn>
```

Comment:
@jmarolf Could you open an issue so that we turn it back on when the [fix](https://github.com/dotnet/sdk/pull/1188) is in the product